cypress/support/step_definitions/constraint-creation.js

```diff
@@ -70,8 +70,8 @@ when("I select patients that are part of study Oracle_1000_Patient with age betw
   cy.get('input[placeholder="max:65"]').type('55');
   cy.contains('numerical_1 ').trigger('dragstart');
   cy.get('input[placeholder="add criterion"]').eq(1).trigger('drop');
-  cy.get('input[placeholder="min:-5.76679"]').type('0');
-  cy.get('input[placeholder="max:29.79301"]').type('10');
+  cy.get('input[placeholder="min:5.766787251"]').type('0');
```

Comment:
test failing

Reply:
another failing test: 

> create a constraint with concept negation

Reply:
looks like transmart-dev not returning the right response

Reply:
yes, this is what I mentioned in the description of this PR -> back-end issue addressed in TMT-778